### PR TITLE
Issue 182 draft

### LINKED ISSUE #627: [182] Issue #182 - April 08, 2021
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/main/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/main/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

### PR DESCRIPTION
**Pull request by fassko**
Closes #627

### CODE REVIEW

File: _drafts/2021-04-15-issue-182.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+{% include sponsor.html %}
+
+### Starter tasks
+
+> [SR-14443](https://bugs.swift.org/browse/SR-14443) [Compiler] [C++] Don't crash on invalid dependent values. 
+
+### Podcasts
+
+> TODO: Latest episode(s) of Swift Unwrapped & the Swift Community Podcast
```
Comment (fassko) [inline _drafts/2021-04-15-issue-182.md:30]:
```suggestion
```

Can remove if there is nothing. Similarly with other sections with TODO

File: _drafts/2021-04-15-issue-182.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+[SE-0308](https://github.com/apple/swift-evolution/blob/main/proposals/0308-postfix-if-config-expressions.md) *`#if` for postfix member expressions* is [under review](https://forums.swift.org/t/se-0308-if-for-postfix-member-expressions/47163/2)
+
+[SE-0292](https://github.com/apple/swift-evolution/blob/main/proposals/0292-package-registry-service.md) *Package Registry Service* [under 2nd review](https://forums.swift.org/t/se-0292-2nd-review-package-registry-service/46917)
+
+> Based on the feedback from the [first review](https://forums.swift.org/t/se-0292-package-registry-service/), the core team feels confident that the ideas behind Package Registry Service are useful and put the Swift packages ecosystem on the right path. One key area the core team asked to further explore in response to the first review, was the nature of the package identifiers given their potential utility in addressing Swift module name conflicts.
+
+### Swift Forums
+
+> TODO
```
Comment (fassko) [inline _drafts/2021-04-15-issue-182.md:92]: We can add here this post https://forums.swift.org/t/allowing-unknown-in-arbitrarily-nested-patterns/47224

**Review by fassko — COMMENTED** [review]
Nice job @jeroenleenarts ! 

Remove empty sections with TODO and add this one forum thread, actually so many Swift language proposal changes.

**Review by JaredHalpern — APPROVED** [review]
I added some small grammatical touch-ups. Looks great!

**Comment by jeroenleenarts** [comment]
@morqon @fassko When both of you agree to this pull, I'd say let's go. We landed a ton of new subscribers due to the mention by Antoine in his newsletter. 👍

**Review by morqon — CHANGES_REQUESTED** [review]
I've added my copy edits in [the usual PR](https://github.com/SwiftWeekly/swiftweekly.github.io/pull/630), hope they're useful!

**Comment by jeroenleenarts** [comment]
Work by @morqon and @JaredHalpern is in. So @fassko What's next?

**Comment by fassko** [comment]
@jeroenleenarts merge the PR. Check locally if everything is fine. Then move the from _drafts to _posts. You can do it in master no need for PR. Then we can send it out. ;)

**Comment by jeroenleenarts** [comment]
Will do that in about an hour. 👍

**Comment by fassko** [comment]
Congrats @jeroenleenarts with your first Swift Weekly Brief issue! 👍🚀

Thank you @JaredHalpern and @morqon for your edits. I love that we are more now! 🙃

**Comment by JaredHalpern** [comment]
Nice work, @jeroenleenarts!

**Pull request merged**